src/index.html

```diff
@@ -10,13 +10,110 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="top-navigation">
+        <img
+          src="./images/logo.png"
+          alt="Moyo"
+          class="top-logo"
+          width="40px"
+          height="40px"
+          loading="lazy"
+        />
+
+        <ul class="top-navigation-list">
+          <li class="top-navigation-item">
+            <a
+              href="#"
+              class="top-navigation-link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="top-navigation-item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="top-navigation-item">
```